weather-app-orig/js/main.js

```diff
@@ -0,0 +1,68 @@
+/* Javascript goes here! */
+
+$(document).ready(function(){
+
+
+  $('.getWeatherData').submit(function(event){
+    event.preventDefault();
+    //we have to prevent the default event so the page doesn't
+    //refresh every time it loads.
+    requestWeatherData();
+
+    //NOTE: So, how is it that you can use the dataType to get the name from the
+    //obj? /\(--__--)/\
+
+  })
+})
+
+function requestWeatherData(myCity){
+  //TODO: Code me
+  //Okay now I get it. So requestWeatherData literally just means put AJAX here.
+  //Could that have been better explained earlier?
+  $.ajax({
+    //The method we post or receive
+    method: 'GET',
+    //A link to the source
+    url: 'http://api.openweathermap.org/data/2.5/weather',
+    //What data type are we using?
+    dataType: 'json',
+    data: $('form').serialize(),
+    //What do we want to do if it succeeds or fails?
+    success: onSuccess,
+    error: onError
+  })
+  //console.log('here is the city ', myCity);
+  //console.log('Testing requestWeatherData function');
+}
+
+function calculateTempurature(myCity){
+  //k is placement for now
+  console.log(myCity, ' is my city')
+  var k = myCity.main.temp;
+  var temp = Math.floor(9/5 * (k - 273) + 32);
```

Comment:
Instead of doing all the math, you could have also used the API which requires another parameter to be sent as data in your ajax call 'units=imperial'. Since you are serializing your form to get the data automatically, you could have added another hidden input for this `<input type="hidden" name="units" value="imperial">`.